src/chrome/content/rules/RWTH-Aachen.xml

```diff
@@ -0,0 +1,145 @@
+	<target host="publications.rwth-aachen.de" />
+	<target host="sharepoint.ecampus.rwth-aachen.de" />
+	<target host="svn.rwth-aachen.de" />
+	<target host="tedx.rwth-aachen.de" />
+	<target host="ths.rwth-aachen.de" />
+	<target host="www-i6.informatik.rwth-aachen.de" />
+	<target host="www.academy.rwth-aachen.de" />
+	<target host="www.ada.rwth-aachen.de" />
+	<target host="www.afu.rwth-aachen.de" />
+	<target host="www.aices.rwth-aachen.de" />
+	<target host="www.akustik.rwth-aachen.de" />
+	<target host="www.amr.rwth-aachen.de" />
+	<target host="www.anglistik.rwth-aachen.de" />
+	<target host="www.animation.rwth-aachen.de" />
+	<target host="www.aso.rwth-aachen.de" />
+	<target host="www.asta.rwth-aachen.de" />
+	<target host="www.avt.rwth-aachen.de" />
+	<target host="www.bioac.ac.rwth-aachen.de" />
+	<target host="www.biologie.rwth-aachen.de" />
+	<target host="www.biotec.rwth-aachen.de" />
+	<target host="www.bur.rwth-aachen.de" />
+	<target host="www.chemie.rwth-aachen.de" />
+	<target host="www.cmm.rwth-aachen.de" />
+	<target host="www.cwd.rwth-aachen.de" />
+	<!-- <target host="www.dwi.rwth-aachen.de" /> -->
+	<target host="www.e3d.rwth-aachen.de" />
+	<target host="www.ecpe.rwth-aachen.de" />
+	<target host="www.elektrotechnik.rwth-aachen.de" />
+	<target host="www.eme.rwth-aachen.de" />
+	<target host="www.euv.rwth-aachen.de" />
+	<target host="www.ezw.rwth-aachen.de" />
+	<target host="www.fb1.rwth-aachen.de" />
+	<target host="www.fb3.rwth-aachen.de" />
+	<target host="www.fb5.rwth-aachen.de" />
+	<target host="www.fgeo.rwth-aachen.de" />
+	<target host="www.filmstudio.rwth-aachen.de" />
+	<target host="www.fset.rwth-aachen.de" />
+	<target host="www.fsmb.rwth-aachen.de" />
+	<target host="www.germlit.rwth-aachen.de" />
+	<target host="www.ghi.rwth-aachen.de" />
+	<target host="www.global-assess.rwth-aachen.de" />
+	<target host="www.graphics.rwth-aachen.de" />
+	<!-- <target host="www.hci.rwth-aachen.de" /> -->
+	<target host="www.iamb.rwth-aachen.de" />
+	<target host="www.ibac.rwth-aachen.de" />
+	<target host="www.ice.rwth-aachen.de" />
+	<target host="www.ict.rwth-aachen.de" />
+	<target host="www.ifam.rwth-aachen.de" />
+	<target host="www.igad.rwth-aachen.de" />
+	<target host="www.igpm.rwth-aachen.de" />
+	<target host="www.ihf.rwth-aachen.de" />
+	<target host="www.ika.rwth-aachen.de" />
+	<target host="www.ima-zlw-ifu.rwth-aachen.de" />
+	<target host="www.immo.rwth-aachen.de" />
+	<target host="www.inets.rwth-aachen.de" />
+	<target host="www.irt.rwth-aachen.de" />
+	<target host="www.isb.rwth-aachen.de" />
+	<target host="www.isf.rwth-aachen.de" />
+	<target host="www.isl.rwth-aachen.de" />
+	<target host="www.ita.rwth-aachen.de" />
+	<target host="www.ithe.rwth-aachen.de" />
+	<target host="www.iw.rwth-aachen.de" />
+	<target host="www.iwe1.rwth-aachen.de" />
+	<target host="www.iwomp2015.rwth-aachen.de" />
+	<target host="www.kawo2.rwth-aachen.de" />
+	<!-- <target host="www.kbsg.rwth-aachen.de" /> -->

```

Comment:
Please document the issue with the commented hosts.
You should list all of them in one single comment block on the top.